### DISCUSSION TITLE
How does it come that "This issue has been deleted" by "me" but I were offline/sleeping at this time?

### DISCUSSION THREAD

**Discussion by thomasmerz** [upptime/upptime#239] [Q&A] [answered]
### Description of my problem:
One of my checked services was "degraded" tonight and Upptime was sending a notification to my Issues 👍🏻 
```
[master e2a33d3] 🟨 Pihole-KA is degraded (200 in 4381 ms) [skip ci] [upptime]
 1 file changed, 3 insertions(+), 3 deletions(-)
e2a33d310e7fb302683e34683108b889dfc7a7c4
Status is different up to degraded
Found 0 issues
Opened and locked a new issue
Sending notification 🟨 Pihole-KA ($URL_PIHOLE_MERZ_NIMBUS) is experiencing **degraded performance**: https://github.com/thomasmerz/upptime/issues/25
```
Action: https://github.com/thomasmerz/upptime/runs/1799953510?check_suite_focus=true#step:3:58
Commit: https://github.com/thomasmerz/upptime/commit/e2a33d310e7fb302683e34683108b889dfc7a7c4
Issue: https://github.com/thomasmerz/upptime/issues/25

**But** Issue says 😞 
"**This issue has been deleted.**"
![image](https://user-images.githubusercontent.com/18568381/106382230-8585b300-63be-11eb-8234-a41fed47203a.png)
(Only I will see this information says the lock)

**Comment by thomasmerz** [comment]
### **Questions to Community or @AnandChowdhary ❓**
* So **how did it come**, that "I" have deleted this issue while I was sleeping?
* **Which automatism** does this?
* Or is this a **bug** (I used "assignees" in .upptimerc.yaml for the first time - before this I got issues, but no notification by GitHub because I wasn't assigned to me)?


For my "luck" I have this a script running by crontab (with parameter "down") so I got notified about this… Without this I would never been noticed this… 🤔 
```
#!/bin/bash

# this script shows UP or DOWN URLs
# from my "Upptime" Monitoring on GitHub Page
# and sends an email for DOWN URLs:

case $1 in
  up)
    echo "Querying https://thomasmerz.github.io/upptime/" for status $1
    OUT=$(curl "https://raw.githubusercontent.com/thomasmerz/upptime/master/history/summary.json" 2>/dev/null | \
      jq '.[] | select(.status == "up")')
    [ ! -z "$OUT" ] && printf '%s\n' "$OUT"|jq || echo "No results for status $1"
    ;;
  down)
    echo "Querying https://thomasmerz.github.io/upptime/" for status $1
    OUT=$(curl "https://raw.githubusercontent.com/thomasmerz/upptime/master/history/summary.json" 2>/dev/null | \
      jq '.[] | select(.status != "up")')
    # send email if OUT is not empty:
    [ ! -z "$OUT" ] && \
      printf '%s\n' "$OUT"|tee /dev/tty| \
      mail -s "Upptime: ⚠️ Degraded performance or 🚨 Complete outage - Details: https://thomasmerz.github.io/upptime/" thomas@mymaildomain.de || \
        echo "No results for status $1"
    ;;
  *)
    echo "Please specify '$0 up' or '$0 down'."
    exit 1
    ;;
esac
```

**Comment by AnandChowdhary** [comment, marked as answer]
Sometimes, we get a false positive. For example, a site is down one check and immediately is up in the next, we consider this a false positive, because chances are that the site wasn't really down. When this happens, we automatically delete the opened issue. If we don't do this, your uptime monitor may get flooded with false positives. Since we use your personal access token to delete issues, GitHub displays that as you deleting that issue.

You can use the configuration option `skipDeleteIssues` to disable this feature.

**Reply by thomasmerz** [reply]
Seems legite 👍🏻 (that @upptime-bot deletes the issue with my personal-access-token (valid for this repo))

But… what about so many or so long "degraded" states "in a row"? Are these really "false positives"? 🤔 
<img src="https://user-images.githubusercontent.com/18568381/106388241-ec669480-63dd-11eb-9120-d28e0d9f1135.png" width="300">
